### PR TITLE
Server qol

### PR DESCRIPTION
**Pull request by TranquillyUnpleasant**
Some QoL changes to the server terminal interface. Changes:
- `maps` displays custom maps only by default.
- Remove colors from map names so the name is actually readable for some maps.
The command has an additional `all` optional parameter to show default maps as well. 
The command description has been updated to reflect this.

- Color codes are removed from various places that show map names in terminal.
- Map finding function now strips both input and map name equally before comparing, making it compatible with all input/map name combinations. This change is reflected in both `host` and `nextmap` commands.

- The `help` command has an additional optional command parameter, to get help on a specific command.
The command description for help has also been updated to reflect this change.

- Look ma im a contributor!

### CODE REVIEW

**Comment by TranquillyUnpleasant** [comment]
P.S. might want to look at the individual commits for less headache.

File: server/src/mindustry/server/ServerControl.java [hunk 1]
```diff
@@ -259,7 +259,17 @@ protected void setup(String[] args){
     }
 
     protected void registerCommands(){
-        handler.register("help", "Displays this command list.", arg -> {
+        handler.register("help", "[command]", "Display the command list, or get help for a specific command.", arg -> {
+            if (arg.length > 0){
```
Comment (Anuken) [inline server/src/mindustry/server/ServerControl.java:263]: incorrect formatting similar to this in several places, see guide

File: server/src/mindustry/server/ServerControl.java [hunk 2]
```diff
@@ -259,7 +259,17 @@ protected void setup(String[] args){
...
+                Command command = handler.getCommandList().find(c -> c.text.equalsIgnoreCase(arg[0]));
+                if (command == null) {
+                    err("Command " + arg[0] + " not found!");
+                }else{
+                    info(command.text + ":");
+                    info("  &b&lb " + command.text + (command.paramText.isEmpty() ? "" : " &lc&fi") + command.paramText + "&fr - &lw" + command.description);
+                }
+                return;
```
Comment (Anuken) [inline server/src/mindustry/server/ServerControl.java:271]: use an else here instead, a return would be confusing

File: server/src/mindustry/server/ServerControl.java [hunk 3]
```diff
@@ -335,14 +345,17 @@ protected void registerCommands(){
             }
         });
 
-        handler.register("maps", "Display all available maps.", arg -> {
+        handler.register("maps", "[all] [full]", "Display available maps. 'all' to include default maps. 'full' to show full map names.", arg -> {
+            boolean all = Arrays.asList(arg).contains("all");
+            boolean full = Arrays.asList(arg).contains("full");
```
Comment (Anuken) [inline server/src/mindustry/server/ServerControl.java:350]: I see no reason to display full map names in the console.

File: server/src/mindustry/server/ServerControl.java [hunk 4]
```diff
@@ -335,14 +345,17 @@ protected void registerCommands(){
             }
         });
 
-        handler.register("maps", "Display all available maps.", arg -> {
+        handler.register("maps", "[all] [full]", "Display available maps. 'all' to include default maps. 'full' to show full map names.", arg -> {
+            boolean all = Arrays.asList(arg).contains("all");
```
Comment (Anuken) [inline server/src/mindustry/server/ServerControl.java:349]: This is a bizarre way to handle arguments, `all` should be a boolean argument with false as a default, true if there's an argument and `arg[0].equals("true")`

**Review by Anuken — CHANGES_REQUESTED** [review]

File: server/src/mindustry/server/ServerControl.java (hunk 1 above)
Comment (TranquillyUnpleasant) [reply server/src/mindustry/server/ServerControl.java]: Fixed

**Comment by TranquillyUnpleasant** [comment]
Requested changes have been made.

**Pull request merged**